evm/src/main/java/org/hyperledger/besu/evm/gascalculator/FrontierGasCalculator.java

```diff
@@ -239,6 +240,31 @@ public long callOperationGasCost(
       final Wei transferValue,
       final Account recipient,
       final Address to) {
+    return callOperationGasCost(
+        frame,
+        stipend,
+        inputDataOffset,
+        inputDataLength,
+        outputDataOffset,
+        outputDataLength,
+        transferValue,
+        recipient,
+        to,
+        true);
```

Comment:
why default to true?

Reply:
The notion of charging for "cold storage" and "cold accounts" was added in Berlin, so the default would be to get the price without this cold access surcharge.  For all calculators pre-berlin it's a distinction without difference, however, as all prices are the same.